string_manipulation.rb

```diff
@@ -22,7 +22,7 @@ def string_reverse(my_string)
 def reverse_words(my_words)
   i = 0
   i2 = 0
-  my_words = my_words.split('')
+  my_words = my_words.split('') # converting into a literal array of characters to allow for modification

```

Comment:
Same comment as above. Eliminate the use of split. Using split and join creates and uses more space behind the scenes.